### DISCUSSION TITLE
404 - since last update

### DISCUSSION THREAD

**Discussion by Daniel-WW** [dani-garcia/vaultwarden#4311] [Q&A]
Hello everyone,

I run vaultwarden in my docker stack, with traefik v2 as a reverse proxy. For two days - I think since the last update - I can no longer open the web interface and get a 404.

I tried to check the settings but I can't understand what the problem is. I have switched the logs to debug mode, but I don't get any information about web access.

I'm quite a beginner at these things and I don't really know where to start troubleshooting. I would be happy about any help and I can post further information.

Thank you in advance - Daniel

**Comment by BlackDex** [comment]
If you do not get logs about access, it must be traefik in some way which can't connect to Vaultwarden.

There were also some issues with a docker version, so check that you are not on v25.0.0 but a version older or newer.

I'm no expert on Traefik, so can't help you there.

**Reply by Daniel-WW** [reply]
Hi, thanks for your fast response. - Yes, I guess it is a problem with the traefik settings. I didn't change anything. If someone else has experience with Traefik, that would be a great help.

**Reply by Pracium** [reply]
I had an similar issue, updating docker fixed it for me.
